style.css

```diff
@@ -427,6 +428,22 @@
             background: none !important;
         }
     }
+    
+    if (hideAuthorBarOnThumbnails) {    
```

Comment:
Co z obrazkami bez białego paska (kiedy uploadujący ma wyłączoną tę opcję w ustawieniach Wykopu)? Czy takie zdjęcie również zostanie przycięte?

Reply:
Tak, dlatego jako opcja konfiguracyjna, domyślnie wyłączona

Reply:
Dla mnie będzie to niestety traktowane jako wadliwa opcja, ucinająca obrazki, które nie miały belki. Dopóki nie znajdziemy sposobu, aby odróżnić obrazki z belką o tych bez, aby poprawnie tę opcję zaimplementować to myślę że to nie ma sensu. Ale dziękuję za pomysł i za pomoc.